static/docs/user-guide/contributing-documentation.md

```diff
@@ -0,0 +1,113 @@
+# Contributing Documentation
+
+We welcome contributions to [DVC.org](https://github.com/iterative/dvc.org) by
+the community.
+
+## How to report a problem
+
+Please search [issue tracker](https://github.com/iterative/dvc.org/issues)
+before creating a new issue. This issue can be about a typographical error, an
```

Comment:
What do you think about removing:
```
This issue can be about a typographical error, an
updation of the documentation or simply an improvement in generation of
documentation.
```

I don't think it adds a lot, the header is about `How to report a problem`, maybe it is better to change the header and rephrase the paragraph

Reply:
@mroutis It just tells the user what all types can a problem be of.

Reply:
Well. If you are leaving this one, correct the following:
- `an updation` is not an English word, maybe you want to use `updating the documentation` instead.
- Please, remove the `simply`, it is a _filler_ that could be interpreted wrong (because improving the way we are generating our docs might be not simple).
- `improvement in generation of documentation` to `or a general improvement`.

I would suggest something like the following:
```
This issue can report a typographical error, the use of better wording, or any other improvement
```

Reply:
@mroutis Will make this change it the separate PR.

Reply:
Thanks @mroutis for a valuable feedback!

Reply:
@algomaster99 a separate PR is totally fine (or include this change into another PR).